### ISSUE TITLE
Path/Name copy options ( Feature request )

### ISSUE THREAD

**Issue by ghost** [Osmose/advanced-open-file#156]
Would be awesome to be able to copy the path of a highlighted file while navigating without having to open the file itself. Simply navigate around, copy an image files ( Relative path ) for example and then exit out and paste that into your code.

This would be great for web development for those who do not want to install the ide-html etc.. packages.

It could of course have countless other uses.

One package that implements this functionality already is 
https://github.com/msansoni/atom-path-copy

It has the options to choose many dif types of paths, file names etc.. all of this functionality via the navigation offered by Advanced open file would make for a great combo.

Would be nice to have options to right click and select copy options similar to the atom-path-copy from within the Advanced open file window as well as be able to set keybinds to do it directly from withing the advanced-open-file window.

To take it  even a step further than atom-path-copy, it would be amazing if you could copy the relative path to the file based on the active pane you are in when you activate advanced-open-file.. meaning you could even copy paths up folders like ../../css/styles.css for example. 

In atom-path-copy he is calling "Relative path" based on the actual "Project path" and doing something weird for his "Project path" I would scrap his style project path, use "Project path" for the actual project path based on the root of the project, then have a relative path feature as described just above.

**To sum it up, these would be the features requested**

- Copy file name ( does not include type: example: test ) _[ Ctrl-alt-n]_
- Copy file full name ( includes type: example: test.scss ) _[ Ctrl-alt-f]_
- Copy full path ( path based on the desktop or OS root folder ) _[ Ctrl-alt-a]_
- Copy relative path ( based on the pane that is active when you activate advanced-open-file ) _[ Ctrl-alt-r]_
- Copy project path ( based on the location of the projects root folder ) _[ Ctrl-alt-p]_
- It would also be nice if you could choose the trailing slash used, since windows uses \ as default and in some cases a / is required or desired. ;-)

Just a thought for a feature that I think would make advanced-file-open even more amazing!

Thanks for reading!
Slater

**Comment by Osmose** [comment]
Thanks for filing the issue!

I think a keyboard shortcut for copying the full path of the highlighted file is reasonable. The relative path from project root also seems reasonable. The others I'm less interested in.

I won't be working on it but would happily review a patch adding these two shortcuts.
